src/enclave/enclave_event_channel.c

```diff
@@ -145,18 +150,36 @@ void initialize_enclave_event_channel(
 
     evt_chn_lock = (struct ticketlock**)oe_calloc(
         evt_channel_num, sizeof(struct ticketlock*));
+    if (!evt_chn_lock)
+    {
+        sgxlkl_fail("Could not allocate memory for evt_chn_lock\n");
```

Comment:
I wonder if we should add a checked-malloc macro. At least in src/enclave we always abort anyways and it would make the code more compact.

Reply:
Yeah, David suggested the same. Where would we put it, as we don't want to edit the OE headers?

Reply:
Maybe `enclave_util.h`?

Reply:
Done.